tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/OrganisationControllerTests.cs

```diff
@@ -146,12 +236,35 @@ public async Task GetIdByAsync_OrganisationExists_ReturnsTheOrganisation()
             result.Should().BeOfType<OkObjectResult>();
             var objectResult = result as OkObjectResult;
 
-            objectResult.Value.Should().BeEquivalentTo(new OrganisationViewModel
+            objectResult.Value.Should().BeEquivalentTo(organisation,
+                conf => conf.Excluding(c => c.Id).Excluding(c => c.Address).Excluding(c => c.LastUpdated));
```

Comment:
Can you review the use of excluding the address from the comparison? An address is assigned to the organisation object, should this not be included in the assert?